plan/runner.go

```diff
@@ -164,8 +166,12 @@ func (r *Runner) taskFunc(flowVal cue.Value) (cueflow.Runner, error) {
 		ctx = lg.WithContext(ctx)
 		ctx, span := otel.Tracer("dagger").Start(ctx, t.Path().String())
 		defer span.End()
+		tm := telemetry.Ctx(ctx)
 
 		lg.Info().Str("state", task.StateComputing.String()).Msg(task.StateComputing.String())
+		tm.Push(ctx, event.ActionStarted{
+			Name: taskPath,
+		})
```

Comment:
I have only been following all this tangentially, so this may be an irrelevant comment, but I do wonder if we could just be using the otel trace constructed right above this instead of adding our own system. The `trace` object has an `AddEvent` method and lets you fill out key-value attributes using options.

I realize that this would require buying into the opentelemetry ecosystem, which means it's not feasible in the short term. But if the change here is meant for what we want to do in the long-term, I think there are some advantages to using opentelemetry ([mentioned here](https://github.com/dagger/dagger.io/discussions/278)). Again though, please ignore if this is a huge tangent off what your goal is here, I just saw this in passing and figured I'd mention it.